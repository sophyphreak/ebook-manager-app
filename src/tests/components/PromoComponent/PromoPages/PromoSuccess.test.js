import React from 'react';
import { shallow } from 'enzyme';
import PromoSuccess from '../../../../components/PromoComponent/PromoPages/PromoSuccess';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<PromoSuccess />);
});

test('should render PromoSuccess correctly', () => {
  expect(wrapper).toMatchSnapshot();
});