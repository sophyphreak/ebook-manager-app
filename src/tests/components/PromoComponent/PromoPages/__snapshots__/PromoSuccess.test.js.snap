// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PromoSuccess correctly 1`] = `<SuccessPage />`;
